README.md

```diff
@@ -45,9 +46,19 @@ const config = {
   experienceVersion: 'PRODUCTION',
 }
 
+const searcher = provideHeadless(config);
+
+searcher.setSessionTrackingEnabled(true);
+let sessionId = window.sessionStorage.getItem('sessionId');
```

Comment:
So we're not going to provide an acquireSessionId method for now, and ask people to do this manually?

Reply:
Yep!